wiki/Best_Of/2012/en.md

```diff
@@ -1,50 +1,108 @@
-
-***[Click here to go back to History of osu! 2013/2014](History_of_osu!/4 "wikilink")***
+**_[Click here to return to the Best_Of main page](../Best_Of)_**
```

Comment:
the link should just be `../`
(that means _go back one page_, while the current one says _go back, then go to Best\_Of_)